### PR TITLE
k8s cost post pt2

### PR DESCRIPTION
**Pull request by camerondgray**

### CODE REVIEW

File: posts/2020-04-07-k8s-cost-saving.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+## Run What You Can in Containers
+Along the same lines of running cheaper clusters for development and staging environments you can also save considerable amounts of money by running databases in containers for your non-production environments. For example, if your application requires a Postgres database, running that in a container on your cluster for your development and staging environments and using RDS for production can help keep costs down. At Convox we have implemented resource overlays(need docs link) to help make this possible.
+
+## Scale Up and Scale Down
```
Comment (ddollar) [inline posts/2020-04-07-k8s-cost-saving.md:18]: maybe "Use Only What You Need" or just "Autoscaling" ?

File: posts/2020-04-07-k8s-cost-saving.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+One of the best ways to save money, especially if you have spikey workloads, is to use [autoscaling](https://docs.convox.com/deployment/scaling#autoscaling). All the major clouds have some form of autoscaling support for their Kubernetes clusters though AWS requires you to install additional software to enable it. You can configure your application to autoscale based on simple factors such as CPU and memory utilization and configure your cluster to autoscale instance capacity to meet the needs of the applications installed on it.
+
+## Use the Most Cost Effective Cloud for Your Specific Needs
```
Comment (ddollar) [inline posts/2020-04-07-k8s-cost-saving.md:21]: this title is pretty long.. something shorter? Right Tool for the Right Job  or   Capacity Planning

File: posts/2020-04-07-k8s-cost-saving.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+author: Cameron Gray
+twitter: camerondgray
+tags: [Convox, AWS, EKS, Google Cloud, GCP, Digital Ocean, Azure, Kubernetes, Cloud Costs, Kubernetes Cost]
+description: Some strategies for keeping the cost of running your clusters in check.
+---
+
+Following up on our previous [blog post](https://convox.com/blog/cost-of-running-k8s) about understanding the true costs of running Kubernetes on the various cloud providers the next important question to answer is what can you do to manage your costs? 
+
+
+## A Few Big Clusters is Cheaper Than Many Small Clusters
```
Comment (ddollar) [inline posts/2020-04-07-k8s-cost-saving.md:12]: maybe Economies of Scale?

File: posts/2020-04-07-k8s-cost-saving.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+## Run What You Can in Containers
+Along the same lines of running cheaper clusters for development and staging environments you can also save considerable amounts of money by running databases in containers for your non-production environments. For example, if your application requires a Postgres database, running that in a container on your cluster for your development and staging environments and using RDS for production can help keep costs down. At Convox we have implemented resource overlays(need docs link) to help make this possible.
+
+## Scale Up and Scale Down
+One of the best ways to save money, especially if you have spikey workloads, is to use [autoscaling](https://docs.convox.com/deployment/scaling#autoscaling). All the major clouds have some form of autoscaling support for their Kubernetes clusters though AWS requires you to install additional software to enable it. You can configure your application to autoscale based on simple factors such as CPU and memory utilization and configure your cluster to autoscale instance capacity to meet the needs of the applications installed on it.
```
Comment (beastawakens) [inline posts/2020-04-07-k8s-cost-saving.md:19]: Maybe mention about scaling down staging/similar environments out of hours/weekends? Just to put it in people's heads.

File: posts/2020-04-07-k8s-cost-saving.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+One great benefit for our fellow startups is that almost all the cloud providers have some sort of startup credits available and with seamless cloud portability you have the option of maximizing your use of those credits across providers!
+
+* [AWS Activate](https://aws.amazon.com/activate/)
+* [Google Cloud For Startups](https://cloud.google.com/developers/startups)
+* [Azure for Startups](https://azure.microsoft.com/en-us/overview/startups/)
+* [DigitalOcean Hatch](https://www.digitalocean.com/hatch/)
+
+Convox is a partner with all four of these providers so if you need help gaining access to credits please [let us know](mailto:sales@convox.com)!
+
+## Nothing is More Expensive than Your Teams Time
```
Comment (mwarkentin) [inline posts/2020-04-07-k8s-cost-saving.md:34]:
```suggestion
## Nothing is More Expensive than Your Team's Time
```

Comment (camerondgray) [reply posts/2020-04-07-k8s-cost-saving.md:34]: Thank's @mwarkentin good catch. The section on spinning Racks down nights and weekends was inspired by the post your wrote for us years ago: https://convox.com/blog/convox-tips-disabling-racks-evenings-weekends . If you ever want to write another guest post we would love it!

File: posts/2020-04-07-k8s-cost-saving.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+One great benefit for our fellow startups is that almost all the cloud providers have some sort of startup credits available and with seamless cloud portability you have the option of maximizing your use of those credits across providers!
+
+* [AWS Activate](https://aws.amazon.com/activate/)
+* [Google Cloud For Startups](https://cloud.google.com/developers/startups)
+* [Azure for Startups](https://azure.microsoft.com/en-us/overview/startups/)
+* [DigitalOcean Hatch](https://www.digitalocean.com/hatch/)
+
+Convox is a partner with all four of these providers so if you need help gaining access to credits please [let us know](mailto:sales@convox.com)!
+
+## Nothing is More Expensive than Your Team's Time
```
Comment (ddollar) [inline posts/2020-04-07-k8s-cost-saving.md:34]: How about "Time is Money"

**Pull request merged**